src/Files.Backend/ViewModels/Dialogs/AddItemDialog/AddItemDialogViewModel.cs

```diff
@@ -1,65 +1,63 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.ComponentModel;
-using CommunityToolkit.Mvvm.DependencyInjection;
-using Files.Backend.Enums;
-using Files.Backend.Extensions;
-using Files.Backend.Models;
-using Files.Backend.Models.Dialogs;
-using Files.Backend.Services;
 using Files.Shared;
-using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Threading.Tasks;
 
 namespace Files.Backend.ViewModels.Dialogs.AddItemDialog
 {
 	public sealed class AddItemDialogViewModel : ObservableObject
 	{
-		public IImageService ImagingService { get; } = Ioc.Default.GetRequiredService<IImageService>();
+		private readonly IImageService _imagingService = Ioc.Default.GetRequiredService<IImageService>();
```

Comment:
Shouldn't you move `Ioc.Default.GetRequiredService<IImageService>()` to the constructor?

Reply:
Fixed